data/pigui/modules/flight-ui/reticule.lua

```diff
@@ -420,23 +461,289 @@ local function displayManeuverData(radius)
+	["CMD_FLYTO"] = { icon = icons.autopilot_fly_to, tooltip = lui.HUD_BUTTON_AUTOPILOT_FLYING_TO_TARGET },
+	["CMD_FORMATION"] = { icon = icons.autopilot_fly_to, tooltip = lui.HUD_BUTTON_AUTOPILOT_FLYING_TO_TARGET },
+	["CMD_FLYAROUND"] = { icon = icons.autopilot_medium_orbit, tooltip = lui.HUD_BUTTON_AUTOPILOT_ENTERING_ORBIT },
+}
+
+local flightstate_info = {
+	["CONTROL_MANUAL"] = { icon = icons.empty, tooltip = lui.HUD_BUTTON_MANUAL_CONTROL },
+	-- "CONTROL_AUTOPILOT" - depends on the current command
+	-- "CONTROL_FIXSPEED" - depends on the cruise mode
+	["CONTROL_FIXHEADING_FORWARD"] = { icon = icons.prograde_thin, tooltip = lui.HUD_BUTTON_FIX_PROGRADE },
+	["CONTROL_FIXHEADING_BACKWARD"] = { icon = icons.retrograde_thin , tooltip = lui.HUD_BUTTON_FIX_RETROGRADE },
+	["CONTROL_FIXHEADING_NORMAL"] = { icon = icons.normal_thin, tooltip = lui.HUD_BUTTON_FIX_NORMAL },
+	["CONTROL_FIXHEADING_ANTINORMAL"] = { icon = icons.antinormal_thin, tooltip = lui.HUD_BUTTON_FIX_ANTINORMAL },
+	["CONTROL_FIXHEADING_RADIALLY_INWARD"] = { icon = icons.radial_in_thin, tooltip = lui.HUD_BUTTON_FIX_RADIAL_IN },
+	["CONTROL_FIXHEADING_RADIALLY_OUTWARD"] = { icon = icons.radial_out_thin, tooltip = lui.HUD_BUTTON_FIX_RADIAL_OUT },
+	["CONTROL_FIXHEADING_KILLROT"] = { icon = icons.rotation_damping_on_thin , tooltip = lui.HUD_BUTTON_KILL_ROTATION }
+}
+
+local radial_menu_actions_orbital = {
+	{icon = ui.theme.icons.normal_thin,
+	 tooltip=lc.HEADING_LOCK_NORMAL,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_NORMAL") end},
+	{icon = ui.theme.icons.radial_out_thin,
+	 tooltip=lc.HEADING_LOCK_RADIALLY_OUTWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_RADIALLY_OUTWARD") end},
+	{icon = ui.theme.icons.retrograde_thin,
+	 tooltip=lc.HEADING_LOCK_BACKWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_BACKWARD") end},
+	{icon = ui.theme.icons.antinormal_thin,
+	 tooltip=lc.HEADING_LOCK_ANTINORMAL,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_ANTINORMAL") end},
+	{icon = ui.theme.icons.radial_in_thin,
+	 tooltip=lc.HEADING_LOCK_RADIALLY_INWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_RADIALLY_INWARD") end},
+	{icon = ui.theme.icons.prograde_thin,
+	 tooltip=lc.HEADING_LOCK_FORWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_FORWARD") end},
+}
+
+local function flightAssistButton(pos)
+	local icon_size = flightAssistIconSize
+	local icon_padding = flightAssistButtonPadding
+	local button_size = flightAssistButtonSize
+	local tooltip = ""
+	local back_color = colors.lightBlueBackground:opacity(0.3)
+	-- info:
+	local flightstate = player:GetFlightState()
+	local flightcontrolstate = player:GetFlightControlState()
+	local followtarget = player:GetFollowTarget()
+	local navtarget = player:GetNavTarget()
+	local cmbtarget = player:GetCombatTarget()
+	local frmtarget = player.frameBody
```

Comment:
Is `player.frameBody` an existing bound variable from before this PR? If it's not, it feels a bit weird to have it here as an bound variable instead of a Get*() function.

Reply:
Judging by the code, this attribute is 11 years old.

Reply:
Ah, I just haven't seen it in a while. No action required then! :smile: